### PR TITLE
Fix/boulder collisions

### PR DESCRIPTION
**Pull request by baconeta**
None of these changes are perfect but they are based on actual player feedback for clarity and visual effect

### CODE REVIEW

File: Assets/Scripts/Props/HeavyLift.cs [hunk 1]
```diff
@@ -25,11 +25,11 @@ protected override void OnMouseDown()
             {
                 _liftAnimation.LiftBoulderAnims();
             }
+            
         }
 
         protected override void OnMouseUp()
         {
-            EnableCollisions(true);
```
Comment (TomasMorton) [inline Assets/Scripts/Props/HeavyLift.cs:32]: Oh yeah this is dumb huh. Oops.

Comment (baconeta) [reply Assets/Scripts/Props/HeavyLift.cs:32]: Hehehe

Comment (baconeta) [reply Assets/Scripts/Props/HeavyLift.cs:32]: I feel like we already had this and some change you made must have gotten lost, idk

Comment (TomasMorton) [reply Assets/Scripts/Props/HeavyLift.cs:32]: Nah it was only the bridge that I made better

Comment (baconeta) [reply Assets/Scripts/Props/HeavyLift.cs:32]: Oh I take it all back.

**Pull request merged**